pkgs/servers/eris-go/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildGoModule, fetchFromGitea }:
+
+buildGoModule rec {
+  pname = "eris-go";
+  version = "20230111";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "eris";
+    repo = pname;
+    rev = version;
+    hash = "sha256-lWEdGop90Ld3WvhEXEN0y0CEblsGzD7NfmiYmtkODOk=";
+  };
+
+  vendorHash = "sha256-DDV7LUnGnf24qQ2I9I4MDUx87s1+yDhisVz/Jw4XU6k=";
+
+  postInstall = "cp $out/bin/eris-get $out/bin/eris-put";
+  # eris-get is a multicall binary
+
+  checkPhase = "go test";
```

Comment:
This is already the default `checkPhase`:

```suggestion
```

Reply:
I get this as a default:
```
runHook preCheck
# We do not set trimpath for tests, in case they reference test assets
export GOFLAGS=${GOFLAGS//-trimpath/}

for pkg in $(getGoDirs test); do
  buildGoDir test \"$pkg\"
done

runHook postCheck
```

Reply:
Yes, which still runs the tests all the same (I tested it). Is there a reason you don't want this behavior?

Reply:
Which behavior? I don't see the tests run.

Reply:
The tests are running, I'd prefer more logging but this is fine.

Reply:
More logging would have to be provided by upstream (or you can pass `-v` to `go test`, but we don't do that by default, so inconsistency probably isn't good).